### ISSUE TITLE
shopify.Order.find(fulfillment_status="fulfilled") returns only orders with fulfillment_status=None or fulfillment_status=partial

### ISSUE THREAD

**Issue by ipretender** [Shopify/shopify-fulfillment-integration#7]
shopify.Order.find( fulfillment_status="fulfilled") does not return orders with fulfillment_status="fulfilled" at all.

Instead it does return only None or partial

**Comment by ipretender** [comment]
here is the proof:

[i.attributes['fulfillment_status'] for i in shopify.Order.find(fulfillment_status="fulfilled", limit=250)]
[None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 'partial', None, None, None, 'partial', None, None, None, None, None, None, 'partial', None, None, None, 'partial', None, None, 'partial', None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, 'partial', 'partial', None, None, None, None, None, None, None, None, 'partial', None, None, None, None, None, None, 'partial', None, None, None]

**Comment by kevinhughes27** [comment]
This example is pretty old (sorry about that). I'd recommend checking the API documentation and gem for the latest usage.

**Comment by ipretender** [comment]
"This example is pretty old" - not sure I understand. I don't use any examples.

it seems, Shopify API has the bug. I contacted them on https://ecommerce.shopify.com/c/shopify-apis-and-technology/t/api-bug-in-admin-orders-json-fulfillment_status-fulfilled-467782

if I open a link in any browser 
..../admin/orders.json?fulfillment_status=fulfilled&limit=250
I get orders with fulfillment_status null and partial. 

In conclusion Shopify API the buggiest one I've ever seen

**Comment by kevinhughes27** [comment]
This repo is for an example fulfillment service. This is the repo for our ruby api client: https://github.com/Shopify/shopify_api

This test file might be relevant if something isn't working for you with fulfillments: https://github.com/Shopify/shopify_api/blob/master/test/fulfillment_test.rb

I am going to close this issue since its not related to this code. Please continue the discussion on the forums.
